### ISSUE TITLE
Export Freeform Data generate no Zip file

### ISSUE THREAD

**Issue by scandella** [solspace/craft3-freeform#1538]
### What happened?

Using the new "Export Freeform Data (beta)" tab, the expected export Zip file is never saved.

### Errors and Stack Trace (if available)

```shell
Error: Class "Symfony\Component\Serializer\Context\Normalizer\ObjectNormalizerContextBuilder" not found in /path-to-my-project/dev/vendor/solspace/craft-freeform/packages/plugin/src/Bundles/Backup/Controllers/ExportController.php:154
Stack trace:
#0 [internal function]: Solspace\Freeform\Bundles\Backup\Controllers\ExportController->actionExport()
```

### How can we reproduce this?

1. On the Export Freeform Data (beta)" tab, choose a form to be exported
2. Click on "Begin Export"
3. No Zip file are generated

### Freeform Edition

Pro

### Freeform Version

5.6.2

### Craft Version

Craft Pro 4.12.4.1

### When did this issue start?

After upgrading from older Freeform version

**Comment by scandella** [comment]
Using 5.6.4, the issue remains the same.

**Comment by kjmartens** [comment]
Sorry about this @scandella. I am having a developer check into this issue shortly. 🙂

**Comment by kjmartens** [comment]
Sorry for the delay @scandella...

I can't seem to duplicate this on my end. Out of curiosity, are you attempting to just export a single form? Are you seeing any errors in the browser console? If you try creating a brand new form and export it, does the same error happen?

In the meantime, a developer will see if they can determine anything specific from the error you provided. 🙂

**Comment by scandella** [comment]
Hi,
Thanks for your answer. I've tried with a brand new form and the issue is the same. Exporting one or many forms raises the same error.

Here is the complete error:

[14-Oct-2024 22:05:02 Europe/Paris] An Error occurred while handling another error:
yii\web\HeadersAlreadySentException: Headers already sent in /path-to-my-project/dev/vendor/solspace/craft-freeform/packages/plugin/src/Library/ServerSentEvents/SSE.php on line 32. in /path-to-my-project/dev/vendor/yiisoft/yii2/web/Response.php:369

#0 /path-to-my-project/dev/vendor/yiisoft/yii2/web/Response.php(342): yii\web\Response->sendHeaders()
#1 /path-to-my-project/dev/vendor/yiisoft/yii2/web/ErrorHandler.php(136): yii\web\Response->send()
#2 /path-to-my-project/dev/vendor/craftcms/cms/src/web/ErrorHandler.php(192): yii\web\ErrorHandler->renderException(Object(Error))
#3 /path-to-my-project/dev/vendor/yiisoft/yii2/base/ErrorHandler.php(152): craft\web\ErrorHandler->renderException(Object(Error))
#4 /path-to-my-project/dev/vendor/craftcms/cms/src/web/ErrorHandler.php(66): yii\base\ErrorHandler->handleException(Object(Error))
#5 [internal function]: craft\web\ErrorHandler->handleException(Object(Error))
#6 {main}
Previous exception:
Error: Class "Symfony\Component\Serializer\Context\Normalizer\ObjectNormalizerContextBuilder" not found in /path-to-my-project/dev/vendor/solspace/craft-freeform/packages/plugin/src/Bundles/Backup/Controllers/ExportController.php:154
Stack trace:
#0 [internal function]: Solspace\Freeform\Bundles\Backup\Controllers\ExportController->actionExport()
#1 /path-to-my-project/dev/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#2 /path-to-my-project/dev/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#3 /path-to-my-project/dev/vendor/solspace/craft-freeform/packages/plugin/src/controllers/BaseApiController.php(15): yii\base\Controller->runAction('export', Array)
#4 /path-to-my-project/dev/vendor/yiisoft/yii2/base/Module.php(552): Solspace\Freeform\controllers\BaseApiController->runAction('export', Array)
#5 /path-to-my-project/dev/vendor/craftcms/cms/src/web/Application.php(341): yii\base\Module->runAction('freeform/backup...', Array)
#6 /path-to-my-project/dev/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('freeform/backup...', Array)
#7 /path-to-my-project/dev/vendor/craftcms/cms/src/web/Application.php(309): yii\web\Application->handleRequest(Object(craft\web\Request))
#8 /path-to-my-project/dev/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#9 /path-to-my-project/dev/web/index.php(12): yii\base\Application->run()

**Comment by gustavs-gutmanis** [comment]
Hi @scandella,

It looks like the `ObjectNormalizerContextBuilder` which we are using is only available starting from `symfony/serializer: ^6.1`.

Would it be possible for you to check your `symfony/serializer` version by running 

```
$ composer show symfony/serializer
```

And if it's below `6.1`, could you try updating it to `6.1` in your composer to see if that solves the issue?

```json
{
    "require": {
        "symfony/serializer": "^6.1"
    }
}
```

And then update via `$ composer update symfony/serializer`

If that works, we will update the minimum required version to `6.1`, otherwise we will need to find a better solution.

**Comment by scandella** [comment]
I had symfony/serializer v6.0.19
Upgrading to ^6.1 solved the issue.
Thanks for your help.
